test/e2e/kubevirt.go

```diff
@@ -1988,6 +2005,13 @@ ip route add %[3]s via %[4]s
 				topology: udnv1.NetworkTopologyLayer2,
 				role:     udnv1.NetworkRolePrimary,
 			}),
+			PEntry(nil, testData{
```

Comment:
nit: replace P with X 

Only bother with this if you have to re-push. Functionally, it is the same .